src/Bridge/Symfony/HttpFoundation/StreamedResponseProcessor.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace K911\Swoole\Bridge\Symfony\HttpFoundation;
+
+use Swoole\Http\Response as SwooleResponse;
+use Symfony\Component\HttpFoundation\BinaryFileResponse;
+use Symfony\Component\HttpFoundation\Response as HttpFoundationResponse;
+use Symfony\Component\HttpFoundation\StreamedResponse;
+
+final class StreamedResponseProcessor implements ResponseProcessorInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(HttpFoundationResponse $httpFoundationResponse, SwooleResponse $swooleResponse): void
+    {
+        if (!$httpFoundationResponse instanceof StreamedResponse) {
+            // TODO: Throw exception?
```

Comment:
Yeah, let's throw an exception. Actually you can just use `Assertion::isInstanceOf` like here: https://github.com/k911/swoole-bundle/blob/d1363137d7cd62764fe8107b8d7a5a0cf8c22b72/src/Bridge/Symfony/Messenger/SwooleServerTaskTransportHandler.php#L26

Reply:
done